content/nginx-one/changelog.md

```diff
@@ -34,6 +34,19 @@ Stay up-to-date with what's new and improved in the F5 NGINX One Console.
 
 ## January 20, 2025
 
+### Manage certificates with Config Sync Groups
+
+With the NGINX One Console, you can now:
+
+Manage certificate deployment in Config Sync Groups. You can:
+
+- Add a certificate to a Config Sync Group
+- Remove a deployed certificate from a Config Sync Group
+
+For more information, including warnings about risks, see our documentation on how you can:
+- [Add a file]({{< relref "/nginx-one/how-to/nginx-configs/add-file.md" >}}) 
+- [Manage certificates]({{< relref "/nginx-one/how-to/certificates/manage-certificates.md" >}})
```

Comment:
```suggestion
- [Add a file]({{< ref "/nginx-one/how-to/nginx-configs/add-file.md" >}}) 
- [Manage certificates]({{< ref "/nginx-one/how-to/certificates/manage-certificates.md" >}})
```

Small nits: `relref` has been the historic shortcode we've used, but it's intended for relative references.

Since we give absolute path each time, I've been updating links to to just use `ref` as I go.

Reply:
The last option does not fire a Hugo error when a link is changed: it silently fails if a link breaks.

Instead of a documentation author discovering and taking responsibility for fixing a link if, for example, moving or renaming a bunch of files as part of IA refactoring, it has to be detected by a link checker at a later stage.

Reply:
> it has to be detected by a link checker at a later stage.

I'm not convinced that's a bad thing. After all, we've had to use the CI/CD link checker to find numerous broken links to due changes in external URLs.

And the "later stage" for IA refactoring would happen during a PR.